src/main/java/ui/components/KeyboardShortcuts.java

```diff
@@ -0,0 +1,116 @@
+package ui.components;
+
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCodeCombination;
+import javafx.scene.input.KeyCombination;
+import prefs.Preferences;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * a central place to specify keyboard shortcuts
+ *
+ * Classes that currently have keyboard shortcut code:
+ * ui.components.NavigableListView
+ * ui.issuecolumn.ColumnControl
+ * ui.issuepanel.IssuePanel
+ * ui.MenuControl
+ *
+ * Utility Class:
+ * util.KeyPress
+ */
+public class KeyboardShortcuts {
```

Comment:
currently no, what should happen if a user decides to use the same key for more than one action? should we reset all the shortcuts to the default, just ignore the actions with the same key or reset those particular actions to the default keys?